contracts/src/assertionStakingPool/AssertionStakingPool.sol

```diff
@@ -7,97 +7,54 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "../rollup/IRollupLogic.sol";
-import "../rollup/IRollupCore.sol";
-import "./StakingPoolErrors.sol";
+import "./AbsBoldStakingPool.sol";
+import "./interfaces/IAssertionStakingPool.sol";
 
 /// @notice Staking pool contract for target assertion.
 /// Allows users to deposit stake, create assertion once required stake amount is reached,
 /// and reclaim their stake when and if the assertion is confirmed.
-contract AssertionStakingPool {
+/// Tokens sent directly to this contract will be lost.
```

Comment:
should also state that we assume both the assertion chain and challenge manager will not return more token than the stake, and it is known that excess fund (relative to deposit) would be stuck in this contract forever